spec/support/shared_examples/password_strength.rb

```diff
@@ -94,4 +94,27 @@
     end
     expect(form.submit(password: password)).to eq result
   end
+
+  it 'does not allow a password that does not have the minimum number of graphemes' do
+    user = build_stubbed(:user, email: 'custom@benevolent.com', uuid: '123')
```

Comment:
IMO I'd try to avoid domains that aren't reserved, to avoid references to potential bad places 😬 

```suggestion
    user = build_stubbed(:user, email: 'custom@example.com', uuid: '123')
```

Reply:
copied from a different test, but definitely agree